### ISSUE TITLE
sqrt(2) + sqrt(3) + sqrt(5) raises AssertionError

### ISSUE THREAD

**Issue by andersk** [leovt/constructible#1]
```py
>>> sqrt(2) + sqrt(3) + sqrt(5)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "constructible.py", line 145, in __add__
    a, b = self.join(other)
  File "constructible.py", line 260, in join
    _, f1, f2 = Constructible.join_fields(self.field, other.field)
  File "constructible.py", line 289, in join_fields
    s = f2_base(r)._try_sqrt()
  File "constructible.py", line 336, in _try_sqrt
    n = nn._try_sqrt()
  File "constructible.py", line 342, in _try_sqrt
    result = Constructible(a, self.b / a * Fraction(1, 2), self.r)
  File "constructible.py", line 75, in __init__
    assert isinstance(field, tuple)
AssertionError
```

(Hi! I wrote the [Haskell constructible library](https://hackage.haskell.org/package/constructible-0.1.0.1/docs/Data-Real-Constructible.html).)

**Comment by leovt** [comment]
@andersk Thanks for the bug report. I have added a test reproducing the issue. I will look at it deeper as soon as I have the time.

**Comment by leovt** [comment]
Fixed in Release 0.2
